org.civicrm.reqangular/test/services/api_test.js

```diff
@@ -0,0 +1,122 @@
+define([
+  'common/angular',
+  'common/angularMocks',
+  'common/services/api'
+], function () {
+  'use strict';
+
+  describe('api test', function () {
+    var api, $httpBackend, $rootScope;
+
+    beforeEach(module('common.apis'));
+
+    beforeEach(inject(function (_api_, _$httpBackend_, _$rootScope_) {
+      api = _api_;
+      $rootScope = _$rootScope_;
+      $httpBackend = _$httpBackend_;
+    }));
+
+    describe('sendGET', function () {
+      var entity = 'send_get_entity',
+        action = 'send_get_action',
+        params = {key: 'value'},
+        cache = true,
```

Comment:
you could have a single function encapsulating all the call parameters and the `whenGET` call, would make for a more organized code
```js
function expectAndSendGET(returnValue) {
  var action = 'get', entity = 'DummyEntity';

  $httpBackend
    .whenGET(new RegExp('action=' + action + '&entity=' + entity))
    .respond(returnValue);

  return api.sendGET(entity, action, { filter_name: 'filter_value' }, true);
}
```
```js
beforeEach(function () {
  promise = expectAndSendGET({ /* .. */ });
});
```

also please stick with the real naming convention of entities and actions